.devcontainer/requirements-dev.txt

```diff
@@ -0,0 +1,2 @@
+esbonio
```

Comment:
Is this something we need? It adds a whole number of dependencies into the developer toolchain that we'd have to support going forward.

Reply:
This is the language server for restructured text and as such helpful for writing docs, though I have to admit that it is giving me a bit of trouble. If we want to move mainly to MyST for docs, maybe this is less crucial.

I am not sure I share your concern for future support. This is, after all, only a development environment, so imho it is easier switch things out. In this case, for example, if we think rst is still relevant for now, but we move to MyST later on, we can just drop this.